app/controllers/audits/base_controller.rb

```diff
@@ -1,10 +1,10 @@
 module Audits
   class BaseController < ApplicationController
     layout "audits"
-    helper_method :filter_params, :primary_org_only?
+    helper_method :filter, :filter_params, :primary_org_only?
 
-    def build_filter(override = {})
-      options = {
+    def filter(override = {})
+      @filter_options ||= {
```

Comment:
Not sure about memoizing it. [Sometimes we modify it](https://github.com/alphagov/content-performance-manager/blob/5574c63f8051b70bb88917e1244f7fe7bd26db95/app/domain/audits/find_next_item.rb#L4-L6)

We might need to think in a more robust API for the filter.

Reply:
Yeah fine - I think we should make this immutable in the future. I've undone that change.